packet_flight/js/packet-flight.js

```diff
@@ -10,52 +10,54 @@ var
 packet_counter = {};
```

Comment:
You can remove this line since you included it in `start_animation`. 
May want to initialize it, though.